web/geosearch/datapunt_geosearch/blueprints/engine.py

```diff
@@ -60,7 +63,8 @@ def fetch_data(sourceClass, request_args, datasets, retry=None):
     try:
         response = datasource.execute_queries(datasets=datasets)
     except Exception:
-        _logger.error("Failed to fetch data from {}".format(datasource), exc_info=True)
+        _logger.error("Failed to fetch data from {}".format(datasource),
```

Comment:
Klein dingetje. Bij de logger is het efficienter om geen format te gebruiken, maar de %s (of %r). In dat geval wordt de string alleen geinterpoleerd als het nodig is. Maar, in deze context is het een minor issue.

Reply:
Aanvulling: de %s format zorgt er ook voor dat sentry de melding kan groeperen :-)

Reply:
Thnx. Gefixed.